scripts/studyarea_to_WKT/studyarea_to_WKT.yml

```diff
@@ -0,0 +1,31 @@
+script: studyarea_to_WKT.R
+name: study area geometry to WKT
+description: This script converts the geometry of the study area into a well-known text (WKT) representation.
+author: 
+ - name: Maria Alejandra Molina Berbeo (mmolina@humboldt.org.co)
+ - name: Victor Julio Rincon
+external_link: https://github.com/GEO-BON/biab-2.0
+inputs:
+  studyarea_path:
+    label: study area geometry
+    description: Text string representing the file path to the spatial file containing the spatial boundaries pertaining to the study area. Accepted file types include shapefile (.shp), raster (.tif/GeoTIFF) or  GeoPackage (.gpkg). This path specifies the complete path location of the file within the file system and is utilized for identifying and accessing the spatial boundaries contained within the file.
+    type: text
+    example: "/scripts/studyarea_to_WKT/input/Arauca.gpkg"
+  studyarea_epsg:
+    label: study area EPSG in
```

Comment:
"EPSG of the study area"?

Reply:
i believe that the YML don't accept capital letters  in the first letter, so de description will be "study area EPSG"

Reply:
Hi,
They do accept capital letters for labels. So you can turn it around the way you like :)
It is for the input and output key that snake case is needed. (studyarea_epsg in the example above)

Reply:
done